test/e2e_node/image_id_test.go

```diff
@@ -28,7 +29,7 @@ import (
 
 var _ = framework.KubeDescribe("ImageID [NodeFeature: ImageID]", func() {
 
-	busyBoxImage := "k8s.gcr.io/busybox@sha256:4bdd623e848417d96127e16037743f0cd8b528c026e9175e22a84f639eca58ff"
+	busyBoxImage := imageutils.GetE2EImage(imageutils.BusyBox)
```

Comment:
This test is meant for testing the docker image with sha id instead of numeric tag, so referring the imagename without sha tag will spoil this testcase.

Reply:
right. fixing.